requirements/base.in

```diff
@@ -4,3 +4,4 @@ django-object-actions==0.10.0           # Object actions in Django admin
 djangorestframework-xml==1.3.0
 django-multi-email-field==0.5.1         # Multi email input field in Django admin
 jsondiff==1.1.1                         # Used to detect content metadata changes
+django-fernet-fields==0.5
```

Comment:
@UmanShahzad Is this already in use?

Reply:
Hmm, I had added this because of https://github.com/edx/edx-platform/pull/17927/files#diff-651eb6d7a35f48a733edd7d396b4dbd5R42 -- this PR had been merged earlier, which is when I made this change. We do actually use it, though, do a `git grep fernet` on `master` for this repo -- it's used in many places, but not in `base.txt`, which it should be. Probably just a mistake.

Reply:
`django-fernet-fields` is provided by edx-platform so we don't want to list it in this file. This file is for requirements that are not provided by edx-platform, but that are needed for edx-enterprise code.

Reply:
Right, makes sense. I'm wondering what https://github.com/edx/edx-platform/pull/17927/files#diff-651eb6d7a35f48a733edd7d396b4dbd5R42 is trying to say though -- I'll comment on that PR.

Reply:
I'll just erase this now btw.